scripts/txt_replace.py

```diff
@@ -65,13 +72,17 @@ def fix_dockerfile(version_tuple):
         replace(filename, k, v)
 
 if __name__ == "__main__":
-    nargs = len(sys.argv)
-    if nargs != 4:
-        print("usage: %s VERSION PREVIOUS_VERSION IMAGE_SHA" % path.basename(sys.argv[0]))
-        exit()
+    parser = argparse.ArgumentParser(description="Script for updating the appropriate fields of the CSV",
+                                 formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+    parser.add_argument("-n", "--namespace", help="namespace used for testing")
+    parser.add_argument("version", help="New version of the CSV")
+    parser.add_argument("previous_version", help="Version of CSV being replaced")
+    parser.add_argument("image_sha", help="The image sha of the compiled operator")
+    args = parser.parse_args()
+    config = vars(args)
+    print(config)
 
-    version_tuple = sys.argv[1:]
-    check_version(version_tuple)
+    check_version(config["version"], config["previous_version"])
 
-    fix_csv(version_tuple)
-    fix_dockerfile(version_tuple)
+    fix_csv(**config)
```

Comment:
Tightly coupling the arg names to the function params is convenient but brittle. This is less elegant than what you have but is more maintainable.
```suggestion
    fix_csv(args.version, args.previous_version, args.image_sha, args.namespace)
```

Reply:
done.